apps/full-site-editing/full-site-editing-plugin/full-site-editing/class-full-site-editing.php

```diff
@@ -434,9 +435,28 @@ public function get_close_button_url() {
 		 *
 		 * @param string Current close button URL.
 		 */
-		$close_button_url = apply_filters( 'a8c_fse_close_button_link', $close_button_url );
+		return apply_filters( 'a8c_fse_close_button_link', $close_button_url );
+	}
+
+	/**
+	 * Returns the base URL for the Edit Template Part button. The URL does not contain neither
+	 * the post ID nor the template part ID. Those query arguments should be provided by
+	 * the Template Part on the Block.
+	 *
+	 * @return string edit link without post ID
+	 */
+	public function get_edit_template_part_base_url() {
+		$edit_post_link = esc_url( remove_query_arg( 'post', get_edit_post_link( 0, 'edit' ) ) );
```

Comment:
`esc_url` can be later (see below).

```suggestion
		$edit_post_link = remove_query_arg( 'post', get_edit_post_link( 0, 'edit' ) );
```